Marlin/language_tr.h

```diff
@@ -30,213 +30,214 @@
 #ifndef LANGUAGE_TR_H
 #define LANGUAGE_TR_H
 
+//#define SIMULATE_ROMFONT

```

Comment:
I forget exactly how it functions, but it's been moved to `language.h` (disabled) and has this comment:
```cpp
// Define SIMULATE_ROMFONT to see what is seen on the
// character-based display defined in Configuration.h
```
I see that you have it enabled in `language_kana.h`. Did you find it necessary to have it enabled there for kana to work?